src/common/core/coretype.js

```diff
@@ -177,117 +177,110 @@ define([
             return node;
         }
 
+        /**
```

Comment:
It should not.
Actually it can be that it will be faster after the change.
Originally, the getChild operation was used to get the overlay information and that can consume resources (as it could end up creating empty core wrapper objects). Now we use the getProperty instead which always returns a json-like object and use less resources.
Node-wise we check roughly the same amount of nodes, or even fewer as now every recursion step gathers the correct 'instantiation sub-tree'.